pre-docs/bolt_configuration_options.md

```diff
@@ -39,8 +39,12 @@ groups of nodes on the commandline and from plans.
 
 `connect-timeout`: How long Bolt should wait when establishing connections.
 
-`tmpdir`: The directory to upload and execute temporary files on the target.
+`run-as-command`: The command to escalate permissions. The user to escalate to
```

Comment:
We should call out that this must be an array, allowing extra options to be supplied.